lib/matplotlib/_mathtext.py

```diff
@@ -759,16 +754,38 @@ class FontConstantsBase:
     # integrals
     delta_integral = 0.1
 
+    # Percentage of x-height the numerator is shifted up in display style.
+    num1 = 1.4
+
+    # Percentage of x-height the numerator is shifted up in text style if there
+    # is a fraction line.
+    num2 = 1.5
+
+    # Percentage of x-height the numerator is shifted up in text style if there
+    # is no fraction line.
+    num3 = 1.3
+
+    # Percentage of x-height the denominator is shifted down in display style.
+    denom1 = 1.3
+
+    # Percentage of x-height the denominator is shifted down in text style.
+    denom2 = 1.1
+
 
 class ComputerModernFontConstants(FontConstantsBase):
-    script_space = 0.075
-    subdrop = 0.2
-    sup1 = 0.45
-    sub1 = 0.2
-    sub2 = 0.3
+    script_space = 0.1476237
```

Comment:
Can you comment on where these numbers come from?

Reply:
Increased the constants by the same factor the x-height was decreased as a result of calculating the correct x-height, hoping that doing so would minimise the number of changes. (The details are in 70959c36dc7b4f07f15fb5bb82b1c3d054eb4b5d commit message, but yes, I'll be sure to mention the same in a comment as well, since I calculated the multiplying factor incorrectly and will force-push it.)